woodwork/tests/accessor/test_series_accessor.py

```diff
@@ -0,0 +1,702 @@
+    Double,
+    Integer,
+    NaturalLanguage,
+    Ordinal,
+    SubRegionCode,
+    ZIPCode
+)
+from woodwork.utils import import_or_none
+
+dd = import_or_none('dask.dataframe')
+ks = import_or_none('databricks.koalas')
+
+
+def xfail_dask_and_koalas(series):
+    if dd and isinstance(series, dd.Series) or ks and isinstance(series, ks.Series):
+        pytest.xfail('Dask and Koalas Accessors not yet supported.')
+
+
+def test_accessor_init(sample_series):
+    xfail_dask_and_koalas(sample_series)
+    series = sample_series.astype('category')
+    assert series.ww._schema is None
+    series.ww.init()
+    assert isinstance(series.ww._schema, dict)
+    assert series.ww.logical_type == Categorical
+    assert series.ww.semantic_tags == {'category'}
+
+
+def test_accessor_init_with_logical_type(sample_series):
+    xfail_dask_and_koalas(sample_series)
+    series = sample_series.copy().astype('string')
+    series.ww.init(logical_type=NaturalLanguage)
+    assert series.ww.logical_type == NaturalLanguage
+    assert series.ww.semantic_tags == set()
+
+    series = sample_series.copy().astype('string')
+    series.ww.init(logical_type="natural_language")
+    assert series.ww.logical_type == NaturalLanguage
+    assert series.ww.semantic_tags == set()
+
+    series = sample_series.copy().astype('string')
+    series.ww.init(logical_type="NaturalLanguage")
+    assert series.ww.logical_type == NaturalLanguage
+    assert series.ww.semantic_tags == set()
+
+
+def test_accessor_init_with_invalid_logical_type(sample_series):
+    xfail_dask_and_koalas(sample_series)
+    series = sample_series
+    error_message = "Cannot initialize Woodwork. Series dtype is incompatible with NaturalLanguage dtype. " \
+        "Try converting series dtype to string before initializing."
+    with pytest.raises(ValueError, match=error_message):
+        series.ww.init(logical_type=NaturalLanguage)
+
+
+def test_accessor_init_with_semantic_tags(sample_series):
+    xfail_dask_and_koalas(sample_series)
+    series = sample_series.astype('category')
+    semantic_tags = ['tag1', 'tag2']
+    series.ww.init(semantic_tags=semantic_tags, use_standard_tags=False)
+    assert series.ww.semantic_tags == set(semantic_tags)
+
+
+def test_accessor_init_with_name(sample_series):
+    xfail_dask_and_koalas(sample_series)
+    name = 'sample_series'
+    changed_name = 'changed_name'
+
+    use_series_name = sample_series.copy().astype('category')
+    assert use_series_name.name == name
+    assert use_series_name.ww.name == name
+
+    use_input_name = sample_series.copy().astype('category')
+    warning = 'Name mismatch between sample_series and changed_name. Series name is now changed_name'
+    with pytest.warns(ColumnNameMismatchWarning, match=warning):
+        use_input_name.ww.init(name=changed_name)
+    assert use_input_name.name == changed_name
+    assert use_input_name.ww.name == changed_name
+
+
+def test_accessor_init_with_falsy_name(sample_series):
```

Comment:
reminds me that we should be making sure we aren't doing `if self.index` or `if column_name` anywhere. Probably worth walking through the code at some point making sure we didn't introduce it anywhere since I probably haven't been as mindful of it as I should be.